gadgets/trace_sni/test/trace_sni_test.go

```diff
@@ -69,7 +65,7 @@ func TestTraceSNI(t *testing.T) {
 	testContainer := containerFactory.NewContainer(
 		containerName,
 		"while true; do setuidgid 1000:1111 wget --no-check-certificate -T 2 -q -O /dev/null https://inspektor-gadget.io; sleep 0.1; done",
-		containerOpts...,
+		containers.WithContainerImage(containerImage),
```

Comment:
Are all changes related to image really needed?

Reply:
Hi @mauriciovasquezbernal, yes you are right, actually they don't require, I just remembered I was doing some debugging with that and forgot to change it back.